src/index.html

```diff
@@ -13,6 +13,76 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="stars--0">
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+
+      <article class="stars__star"></article>
+    </section>
+
+    <section class="stars--1">
+      <article class="stars--1__star"></article>
```

Comment:
The class name 'stars--1__star' seems to be inconsistent with the class name 'stars__star' used in the same section. Make sure to keep your class names consistent across similar elements.